README.rst

```diff
@@ -57,30 +48,31 @@ To build the documentation on your checkout, simply run::
 
   make docs
 
-We plan to get this published in the near future, and this README will be
-updated when that happens.
-
 Contributions
 *************
 
 All new code should include tests that excercise the code and prove that it
 works, or fixes the bug you are trying to fix.  Any Pull Request without tests
-will not be accepted.
+will not be accepted. See `CONTRIBUTING <CONTRIBUTING.rst>`_ for more details.
+
+Building
+********
+
+If you wish to build a local package for testing at any time, you can simply
+run::
+
+  make dist
+
+this will build a package with a .dev extension that you can install for testing
+and verification.
 
 Pushing a new release
 *********************
 
-1. Prepare a patch to update the version number (`example`_)
+1. Prepare a patch to update the version number (`example`_). Note that we no
+   longer need to update __version__ manually, just the HISTORY.rst file.
 2. Once that's merged, tag that patch and push the new tag to the repo
 3. Github Actions will then automatically build and publish a new release.
 
 
 .. _example: https://github.com/release-depot/git_wrapper/commit/fc88bcb3158187ba9566dad896e3c688d8bc5109
-
-Credits
--------
-
-This package was created with Cookiecutter_ and the `audreyr/cookiecutter-pypackage`_ project template.
-
-.. _Cookiecutter: https://github.com/audreyr/cookiecutter
-.. _`audreyr/cookiecutter-pypackage`: https://github.com/audreyr/cookiecutter-pypackage
```

Comment:
We might want to just say "based on but heavily modified"?

Reply:
I suppose we could keep this, but alter the description.  My thought was that this is so far from how it was initially set up years ago, that what we used to create it at that time was no longer relevant.  If we want to keep it for historical reference, I am ok with that.

Reply:
Ok, so I changed the contributing link to just be test, I am unsure if we have agreed what to do with this part.  If you want it in, I'll add it, otherwise, I think I have addressed the remaining concerns.